airbrus.mt

```diff
@@ -68,6 +68,55 @@ def UTF8JSON :DeepFrozen := composeCodec(UTF8, JSON)
 def makeLineTube() as DeepFrozen:
     return makePumpTube(makeSplitPump(b`$\n`))
 
+def make_giftExchange() :Any as DeepFrozen:
+    def givers  := [].asMap().diverge()
+    def wanters := [].asMap().diverge()
+        
+    def give(giver :Str, acceptor :Str, giftName :Str, gift :Any) :Void :
```

Comment:
Shouldn't this function and the following `accept` function be rolled into the methods which call them further down in the code?

Reply:
Rolled them in. Started out this way due to habit of writing modules in Lua in this style.